src/util/transformThemeValue.js

```diff
@@ -13,15 +13,14 @@ export default function transformThemeValue(themeSection) {
       'transitionTimingFunction',
       'backgroundImage',
       'backgroundSize',
-      'backgroundColor',
       'cursor',
       'animation',
     ].includes(themeSection)
   ) {
     return (value) => (Array.isArray(value) ? value.join(', ') : value)
   }
 
-  if (themeSection === 'colors') {
+  if (['colors', 'textColor', 'backgroundColor', 'borderColor'].includes(themeSection)) {
```

Comment:
Should we do something less explicit here like `themeSection === 'colors' || themeSection.toLowerCase().endsWith('color')`?

Reply:
I think I would prefer to be explicit personally. We are missing a bunch of things in this list though, like `divideColor`, `gradientColorStops`, `placeholderColor`, `ringColor`, `ringOffsetColor`, `fill`, `stroke`, and the brand new `caretColor` plugin so will want to get updated for those.

This is probably going to be annoying to maintain so might be worth thinking through a strategy that is more automatic but I don't have any good ideas that immediately come to mind :/

Reply:
Added the other items for now and alphabetized. Everything I can think of feels either limiting or gross right now. Will think about it a bit more.